.pipelines/.vsts-vhd-builder-release.yaml

```diff
@@ -352,6 +356,31 @@ stages:
         - template: ./templates/.builder-release-template.yaml
           parameters:
             artifactName: marinerv2-gen2
+  - stage: build_vhd_mariner2_gen2_arm64
+    dependsOn: []
+    condition: eq('${{ parameters.buildMarinerV2gen2ARM64 }}', true)
+    jobs:
+    - job: build
+      timeoutInMinutes: 180
+      steps:
+      - bash: |
+          echo '##vso[task.setvariable variable=DRY_RUN]${{parameters.dryrun}}'
+          echo '##vso[task.setvariable variable=OS_SKU]CBLMariner'
+          echo '##vso[task.setvariable variable=OS_VERSION]V2'
+          echo '##vso[task.setvariable variable=IMG_PUBLISHER]MicrosoftCBLMariner'
+          echo '##vso[task.setvariable variable=IMG_OFFER]cbl-mariner'
+          echo '##vso[task.setvariable variable=IMG_SKU]cbl-mariner-2-gen2-arm64'
```

Comment:
The SKU of arm64 mariner 2.0 doesn't have '-gen2'. I double checked with Chris Co before, here is his answer
"Since Azure only supports ARM64 GEN2 VMs, our ARM64 Marketplace image (cbl-mariner-2-arm64) is actually a Gen2 image"


![image](https://user-images.githubusercontent.com/39974812/180562418-8d4992a6-d2dc-4686-bc37-745b84771b9e.png)

Reply:
Thank you for catching this!